packages/pn-personafisica-webapp/src/App.tsx

```diff
@@ -85,44 +84,50 @@ const App = () => {
   }, [sessionToken]);
 
   useEffect(() => {
-    setPendingDelegatorsState(pendingDelegators);
+    if (sessionToken !== '') {
+      void dispatch(getSidemenuInformation());
+    }
   }, [pendingDelegators]);
 
-  const mapDelegatorSideMenuItem = delegators.map((delegator: Delegation) => ({
-    label:
-      'delegator' in delegator && delegator.delegator
-        ? `${delegator.delegator.displayName}`
-        : 'No Name Found',
-    route:
-      'delegator' in delegator && delegator.delegator
-        ? routes.GET_NOTIFICHE_DELEGATO_PATH(delegator.mandateId)
-        : '*',
-  }));
+  const mapDelegatorSideMenuItem = (): Array<SideMenuItem> | undefined => {
+    if(delegators.length > 0) {
+      const myNotifications = {
+        label: t('title', {ns: 'notifiche'}),
+        route: routes.NOTIFICHE
+      };
+      const mappedDelegators = delegators.map((delegator: Delegation) => ({
+        label:
+          'delegator' in delegator && delegator.delegator
+            ? `${delegator.delegator.displayName}`
+            : 'No Name Found',
+        route:
+          'delegator' in delegator && delegator.delegator
+            ? routes.GET_NOTIFICHE_DELEGATO_PATH(delegator.mandateId)
+            : '*',
+      }));
+      return [ myNotifications, ...mappedDelegators];
+    } else {
+      return undefined;
+    }
+  };
 
-  // add your notifications menu item
-  if (mapDelegatorSideMenuItem.length) {
-    /* eslint-disable-next-line functional/immutable-data */
-    mapDelegatorSideMenuItem.unshift({
-      label: t('title', {ns: 'notifiche'}),
-      route: routes.NOTIFICHE
-    });
-  }
+  const sideMenuDelegators = mapDelegatorSideMenuItem();
 
   // TODO spostare questo in un file di utility
   const menuItems: Array<SideMenuItem> = [
     {
       label: t('menu.notifiche'),
       icon: MailOutlineIcon,
       route: routes.NOTIFICHE,
-      children: mapDelegatorSideMenuItem.length ? mapDelegatorSideMenuItem : undefined,
-      notSelectable: !!mapDelegatorSideMenuItem.length
+      children: sideMenuDelegators,
```

Comment:
Dal codice mi sembra che adesso compaia la voce "Le tue notifiche" anche nel caso in cui non si hanno deleghe a proprio carico.
Da specifiche ricordo che il comportamento è diverso:
- niente deleghe, non compare dropdown
- deleghe, compare dropdown con voci di menu "Le tue notifiche" e i nomi di coloro che ci hanno delegato

Se non si hanno avute indicazioni su questo, confrontarsi con Damiano per confermare quanto scritto sopra